src/lib/wordsearch/WordSearchWrapper.ts

```diff
@@ -1,4 +1,4 @@
-import { shuffle, merge, difference, cloneDeep } from "lodash";
+import { shuffle, merge, difference, cloneDeep, map } from "lodash";
```

Comment:
Optimize the imports

Reply:
resolved.
[latest change](https://github.com/SXC-ALCC/Sherlock-Scrambles-Ts/pull/9/commits/80e8c8305f6b159c8f2da9cca316eeef21c50c02)